docs/software-architecture-and-design.md

```diff
@@ -444,46 +530,46 @@ The `AI` is the part of the [Fullsystem](#fullsystem) where all of our gameplay
 The two main components of the `AI` are strategy and navigation.
 
 ## Strategy
-We use a framework called `STP (Skills, Tactics, Plays)` to implement our stratgy. The `STP` framework was originally proposed by Carnegie Mellon University back in 2004. The original paper can be found [here](https://kilthub.cmu.edu/articles/STP_Skills_Tactics_and_Plays_for_Multi-Robot_Control_in_Adversarial_Environments/6561002/1).
+We use a framework called `STP (Skills, Tactics, Plays)` to implement our gameplay strategy. The `STP` framework was originally proposed by Carnegie Mellon University back in 2004. The original paper can be found [here](https://kilthub.cmu.edu/articles/STP_Skills_Tactics_and_Plays_for_Multi-Robot_Control_in_Adversarial_Environments/6561002/1).
 
 `STP` is a way of breaking down roles and responsibilities into a simple hierarchy, making it easier to build up more complex strategies from simpler pieces. This is the core of where our strategy is implemented.
 
-When the [AI](#ai) is given new information and asked to make a decision, our `STP` strategy is what is executed first. It takes in a [World](#world) and returns [Intents](#intents).
-
 ###  STP Diagram
 The STP diagram shows how this works. Functions to assign tactics to robots and build motion constraints are passed into a `Play`'s `get` function, which the `Play` uses to generate tactics with assigned robots and with updated motion constraints.
 
 ![STP Diagram](images/STP.svg)
 
-### Tactics
-The `T` in `STP` stands for `Tactics`. A `Tactic` represents a "single-robots' role" on a team. Examples include:
-1. Being a goalie
-2. Being a passer or pass receiver
-3. Being a defender that shadows enemy robots
-4. Being a defender that tries to steal the ball from enemies
+### Skills
+The `S` in `STP` stands for `Skills`. A `Skill` represents a lower-level behaviour that a robot can execute. Examples include:
 
-They can also represent lower level behaviours, such as
 1. Moving to a position (without colliding with anything)
 2. Shooting the ball at a target
 3. Intercepting a moving ball
```

Comment:
nit: "intercepting" is kind of a subset of "moving" and we don't really treat it as a skill, we could probably replace it with "dribbling the ball"